xcode/Subconscious/Shared/Components/Common/SubtextView.swift

```diff
@@ -38,17 +38,40 @@ struct SubtextView: View {
             return RenderableBlock(block: block, entries: self.entries(for: block))
         }
     }
+    
+    func shouldReplaceBlockWithTransclude(block: Subtext.Block) -> Bool {
+        var count = 0
+        for inline in block.inline {
+            switch (inline) {
+            case .slashlink(let slashlink):
+                count += slashlink.span.count
+                continue
+            case _:
+                continue
+            }
+        }
+        
+        return block.body()
+            .trimmingCharacters(in: .punctuationCharacters)
+            .trimmingCharacters(in: .whitespaces)
+            .count <= count
+    }
```

Comment:
This was a suggestion in Discord from the team, for blocks where there is a slashlink on its own (aside from punctuation and whitespace) then we can replace it entirely with a transclude.

Unsure if this is actually a good idea or not from a principles POV, we've been showing the raw markup _and_ the rendered version so far.

Reply:
Let's go with it and see how it feels in practice. (Aside: this is what I do on my website and it works pretty well)

In future when we have better across the board transclusion support, I want to consider hiding bare links and slashlinks in view mode. This is a good start.